### ISSUE TITLE
[FEATURE] AtlasMap CLI runner

### ISSUE THREAD

**Issue by paoloantinori** [atlasmap/atlasmap#783]
An interesting piece to add to AtlasMap is some generic support to quickly test a mapping generated with AtlasMap designer.

This is still an initial idea that can be discussed further but a simple User Story could resemble the following one:

I'm an AtlasMap user (either standalone or Syndesis), I design my mapping and export the configuration.
I want to easily consume the configuration in a CLI tool, monitoring for example the `in/` folder and automatically put in an `out/` folder the output.
The idea is that a Camel based program brings up all this and a user just needs to drop files in a folder (or pass them at CLI) and check the result.

**Comment by paoloantinori** [comment]
An example of a Camel based java project using AtlasMap can be found here:

http://playintegration.blogspot.com/2019/02/english-check-this-data-mapping-tool.html

**Comment by igarashitm** [comment]
Yep, I've been wishing to have that kind of easy-peasy test utility so long... but Santa hasn't come yet. I think that would be nice to have it as a part of atlasmap-maven-plugin.

**Comment by paoloantinori** [comment]
This has been discussed and a maven plugin doesn't have that immediate user experience that we'd like to offer to users.

**Comment by igarashitm** [comment]
Hi @paoloantinori, can you share about that discussion a bit more here?
* Requirements which maven plugin doesn't satisfy
* Alternatives, plain java like having `atlasmap-cli.jar`?

**Comment by paoloantinori** [comment]
It's really a UX thing.
A typical invocation for Maven plugin has this shape: `mvn com.googlecode.maven-download-plugin:download-maven-plugin:1.3.0:help -Ddetail -Dgoal=wget` and assumes you have both maven and java installed.

We are looking for something more immediate, that might resembles more: `./atlasmap ./input ./output`

I guess that a `java -jar atlasmap-cli.jar` could be acceptable if we cannot go the native way.

bringing @chirino on this conversation in case he wants to share ideas.
